README.md

```diff
@@ -17,273 +17,90 @@
-   download and interact with most of the components of the dataset. **(WIP)**
+The *PLINDER* dataset is provided in two ways:
 
-2. Via raw files on bucket: if you prefer to use the dataset directly, you can fetch it using [`gsutil`](https://cloud.google.com/storage/docs/gsutil) from google cloud storage. **(recommended)**
+- You can either use the files from the dataset directly using your preferred tooling
+  by downloading the data from the public
+  [bucket](https://cloud.google.com/storage/docs/buckets),
+- or you can utilize the dedicated `plinder` Python package for interfacing the data.
 
-If you go with route 2, see below sections.
 
 ## Downloading the dataset
 
-    export PLINDER_RELEASE=2024-06 # Current release
-    export PLINDER_ITERATION=v2 # Current iteration
-    gsutil -m cp -r gs://plinder/${PLINDER_RELEASE}/${PLINDER_ITERATION}/* ~/.local/share/plinder/${PLINDER_RELEASE}/${PLINDER_ITERATION}/
+The dataset can be downloaded from the bucket with
+[gsutil](https://cloud.google.com/storage/docs/gsutil_install):
 
-**❗NOTE**: The current version and the one we plan to use for the MLSB challenge is `gs://plinder/2024-06/v2`, while the version described in the preprint is `gs://plinder/2024-04/v1`.
-
-This yields the following structure, with the `systems`, `splits`, and `index/annotation_table.parquet` being most important for direct usage and the rest containing useful annotations and medadata.
-
-```bash
-2024-06/                     # The "`plinder` release" (`PLINDER_RELEASE`)
-|-- v2                       # The "`plinder` iteration" (`PLINDER_ITERATION`)
-|   |-- systems              # structure files for all systems (split by `two_char_code` and zipped)
-|   |-- splits               # split files and the configs used to generate them
-|   |-- clusters             # pre-calculated cluster labels derived from the protein similarity dataset
-|   |-- entries              # raw annotations prior to consolidation (split by `two_char_code` and zipped)
-|   |-- fingerprints         # index mapping files for the ligand similarity dataset
-|   |-- index                # consolidated tabular annotations
-|   |-- leakage              # leakage analysis results for evaluation sets
-|   |-- ligand_scores        # ligand similarity parquet dataset
-|   |-- ligands              # ligand data expanded from entries for computing similarity
-|   |-- linked_structures    # linked apo and predicted structure files
-|   |-- links                # parquet files linking systems to apo and predicted structures
-|   |-- mmp                  # ligand matched molecular pairs (MMP) and series (MMS) data
-|   |-- scores               # protein similarity parquet dataset
+```console
+$ export PLINDER_RELEASE=2024-06 # Current release
+$ export PLINDER_ITERATION=v2 # Current iteration
+$ gsutil -m cp -r gs://plinder/${PLINDER_RELEASE}/${PLINDER_ITERATION}/* ~/.local/share/plinder/${PLINDER_RELEASE}/${PLINDER_ITERATION}/
```

Comment:
I would add a statement after this that says something like "For details on the description of sub-directories, see ..."